src/Tabs.jsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module TabsListTab
+ *
+ * Direct child of `TabsList`.
+ * Renders individual tab control.
+ * Accepts a `selected` prop.
+ */
+export class TabsListTab extends React.Component {
+	render() {
+		const {
+			children,
+			className,
+			selected,
+			onClick,
+			tabsRef,
+			tabsIndex,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'tabs-tab',
+			{
+				'tabs-tab--selected': selected
+			},
+			className
+		);
+
+		return (
+			<li
+				role='tab'
+				id={`${tabsRef}_tab_${tabsIndex}`}
+				aria-controls={`${tabsRef}_panel_${tabsIndex}`}
+				onClick={onClick}
+				className={classNames}
+				{...other}>
+				{children}
+			</li>
+		);
+	}
+}
+TabsListTab.propTypes = {
+	selected: React.PropTypes.bool,
+	onClick: React.PropTypes.func,
+};
+TabsListTab.defaultProps = {
+	selected: false,
+};
+
+/**
+ * @module TabsList
+ * Direct child of `Tabs`.
+ * Renders a `nav` and `ul` to contain tab conrols
+ */
+export class TabsList extends React.Component {
+	renderChildren() {
+		const { tabsRef, children } = this.props;
+
+		return React.Children.map(children, (kid, tabsIndex) => {
+			return React.cloneElement(kid, { tabsRef, tabsIndex });
+		});
+	}
+	render() {
+		// remove non-standard DOM attributes before render returns
```

Comment:
yeah you can always tell eslint to ignore that rule for those lines, though: http://eslint.org/docs/user-guide/configuring.html#configuring-rules